DEVELOPING.md

```diff
@@ -1,22 +1,90 @@
-## Generate python types from Cosmos-SDK protobuf schemas
+## Development setup
 
-### Fetch Cosmos-SDK protobuf schema files
->NOTE: When updating to a different version of the Cosmos SDK (see the version currently used under `COSMOS_SDK_VERSION` in [Makefile](#Makefile])), you will need to perform this step once to fetch the relevant protobuf schema files:
+The easiest way to get set up for development is to install Python `>=3.7` and `pipenv`, then run the following:
```

Comment:
We should make sure that we support these versions of Python. It looks like that now we require Python 3.9.

```
[requires]
python_version = "3.9"
```

Reply:
Removed all occurrences of enforcing just 3.9